scanner/yara/compile.go

```diff
@@ -110,13 +110,25 @@ func compile(what int, inputfiles []string) (*yr.Rules, error) {
 			return nil, err
 		}
 	}
+	whats := [...]string{"file", "process"}[what]
```

Comment:
Name this maybe file_or_process? _what_ is already kind of too generic for naming, but _whats_ won't make it clearer here.

Reply:
Using "what" was a bad idea, too, renamed both.